models.py

```diff
@@ -337,7 +357,8 @@ def is_accessible(self):
             return False
 
         if current_user.has_role('admin'):
-            self.can_export = True
+            return True
```

Comment:
remove this

Reply:
dont want merchants seeing the txnotification table

Reply:
Only admin roles can see the TAB. Also the self.can_export should not be there anymore.